src/io/flutter/view/InspectorPanel.java

```diff
@@ -433,6 +434,10 @@ private MyTree(final DefaultMutableTreeNode treemodel) {
 
       setRootVisible(false);
       registerShortcuts();
+
+      // Decrease indent.
+      final BasicTreeUI treeUI = (BasicTreeUI) getUI();
+      treeUI.setRightChildIndent(3);
```

Comment:
Good find!

Perhaps use something like `JBUI.scale()`? That takes into account HD vs non-HD screens.

Reply:
New screenshot w/ children added.

Reply:
Good call on the scaling.  Fixed.